content/en/docs/pipelines/sdk/OWNERS.md

```diff
@@ -0,0 +1,9 @@
+approvers:
+  - Bobgy
+  - IronPan
```

Comment:
I'd suggest remove @IronPan and me, the other three members have deep knowledge around these.

Reply:
Done it also spotted a typo in reviewers and fixed it ..now should be fine

Reply:
thanks @Bobgy !